src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/Serialization/PropertyVisibilityTests.cs

```diff
@@ -403,10 +403,8 @@ public void PublicContextAndJsonConverter()
         {
             JsonConverter obj = JsonMetadataServices.BooleanConverter;
 
-            string json = JsonSerializer.Serialize(obj, PublicContext.Default.Options);
-            Assert.Equal("{}", json);
-
-            Assert.Throws<NotSupportedException>(() => JsonSerializer.Deserialize<JsonConverter>(json, PublicContext.Default.Options));
+            Assert.Throws<NotSupportedException>(() => JsonSerializer.Serialize(obj, PublicContext.Default.Options));
```

Comment:
Attempting to serialize a JsonConverter will now throw since it carries an unsupported property of type `Type`. I think that's ok since it's not a particularly meaningful scenario.